src/sentry/eventstore/snuba/backend.py

```diff
@@ -336,3 +336,43 @@ def __make_event(self, snuba_data):
         return Event(
             event_id=event_id, group_id=group_id, project_id=project_id, snuba_data=snuba_data
         )
+
+    def get_unfetched_transactions(
+        self,
+        filter,
+        orderby=None,
+        limit=DEFAULT_LIMIT,
+        offset=DEFAULT_OFFSET,
+        referrer="eventstore.get_unfetched_transactions",
+    ):
+        """
+        Get transactions from Snuba, without node data loaded.
+        """
+        assert filter, "You must provide a filter"
+        cols = self.__get_columns()
```

Comment:
This list of columns includes the group id since eventstore was only designed to handle errors. Do you know how that works with the transactions dataset?

Reply:
It gets mapped to None over here https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/events.py#L22-L28